.github/ISSUE_TEMPLATE/c-feature-request.md

```diff
@@ -0,0 +1,24 @@
+---
+
+name: 🚀 Feature Request
+about: Want to see something new included in the Framework? Submit it!
+---
+
+## 🚀 Feature Requests
+
+<!--- Summary description of the feature --->
+
+### Contextualize the feature
+<!--- Where does it fit in the framework and why it's being included there. --->
```

Comment:
Same as above. Maybe replace `framework` for `NuKeeper`?

Reply:
good spot